basic-features.md

```diff
@@ -134,17 +134,18 @@ This document lists the features supported in each TiDB version. Note that suppo
 
 ## Security
 
-| Security                                                 | 6.2 | 6.1 | 6.0 | 5.4  | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
-| ------------------------------------------------------------ | :--: | :--: | :--: | ---- | :-----: | :-----: | :-----: | :-----: | :-----: |
-| [Transparent layer security (TLS)](/enable-tls-between-clients-and-servers.md) | Y | Y | Y | Y    |    Y    |    Y    |    Y    |    Y    |    Y    |
-| [Encryption at rest (TDE)](/encryption-at-rest.md)           | Y | Y | Y | Y    |    Y    |    Y    |    Y    |    Y    |    Y    |
-| [Role-based authentication (RBAC)](/role-based-access-control.md) | Y | Y | Y | Y    |    Y    |    Y    |    Y    |    Y    |    Y    |
-| [Certificate-based authentication](/certificate-authentication.md) | Y | Y | Y | Y    |    Y    |    Y    |    Y    |    Y    |    Y    |
-| `caching_sha2_password` authentication                       | Y | Y | Y | Y    |    Y    |    Y    |    N    |    N    |    N    |
-| [MySQL compatible `GRANT` system](/privilege-management.md)  | Y | Y | Y | Y    |    Y    |    Y    |    Y    |    Y    |    Y    |
-| [Dynamic Privileges](/privilege-management.md#dynamic-privileges) | Y | Y | Y | Y    |    Y    |    Y    |    Y    |    N    |    N    |
-| [Security Enhanced Mode](/system-variables.md#tidb_enable_enhanced_security) | Y | Y | Y | Y    |    Y    |    Y    |    Y    |    N    |    N    |
-| [Redacted Log Files](/log-redaction.md)                      | Y | Y | Y | Y    |    Y    |    Y    |    Y    |    Y    |    N    |
+| Security                                               | 6.3 | 6.2 | 6.1 | 6.0 | 5.4  | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
```

Comment:
Please align with https://github.com/pingcap/docs-cn/pull/10578.
Some commits are not included in this PR.